book/src/configuration.md

```diff
@@ -65,6 +65,7 @@ Its settings will be merged with the configuration directory `config.toml` and t
 | `workspace-lsp-roots` | Directories relative to the workspace root that are treated as LSP roots. Should only be set in `.helix/config.toml` | `[]` |
 | `default-line-ending` | The line ending to use for new documents. Can be `native`, `lf`, `crlf`, `ff`, `cr` or `nel`. `native` uses the platform's native line ending (`crlf` on Windows, otherwise `lf`). | `native` |
 | `insert-final-newline` | Whether to automatically insert a trailing line-ending on write if missing | `true` |
+| `no-break-on-hyphen` | Disable linebreak on hyphen when using `:reflow` | `true` |
```

Comment:
If we do introduce this option it should be named so that it's clear that it's for the reflow command, like `reflow-break-on-hyphen`